src/uu/sort/Cargo.toml

```diff
@@ -25,6 +25,7 @@ memchr = { workspace = true }
 rand = { workspace = true }
 rayon = { workspace = true }
 self_cell = { workspace = true }
+sysinfo = "0.29.10"
```

Comment:
Could you make this a workspace dependency?

Reply:
ok